eureka/S3_data_reduction/nirspec.py

```diff
@@ -59,7 +60,14 @@ def read(filename, data, meta):
                                              data.attrs['intend']]
 
     # Record integration mid-times in BJD_TDB
-    if len(int_times['int_mid_BJD_TDB']) == 0:
+    if (hasattr(meta, 'time_file') and meta.time_file is not None):
```

Comment:
Rather than repeating the code multiple times (once for each instrument), let's turn this code into a function call.